src/content/developers/docs/consensus-mechanisms/pow/index.md

```diff
@@ -3,76 +3,71 @@ title: Proof-of-work (PoW)
 description: An explanation of the proof-of-work consensus protocol and its role in Ethereum.
 lang: en
 sidebar: true
-incomplete: true
 ---
 
-Ethereum, like Bitcoin, currently uses a consensus protocol called **[Proof-of-work (PoW)](https://wikipedia.org/wiki/Proof_of_work)**. This allows the nodes of the Ethereum network to agree on the state of all information recorded on the Ethereum blockchain and prevents certain kinds of economic attacks.
+Ethereum was always intended to be a [proof-of-stake](/developers/docs/consensus-mechanisms/pos) network, but the research and development required to implement proof-of-stake safely, securely and performantly took several years. The network was therefore started with a different consensus mechanism called **[Proof-of-work (PoW)](https://wikipedia.org/wiki/Proof_of_work)**. This allowed the nodes of the Ethereum network to agree on the state of all information recorded on the Ethereum blockchain and prevents certain kinds of economic attacks.
```

Comment:
Do you think the intro talking about proof-of-stake first might be confusing? I feel like we should introduce proof-of-work first (Ethereum originally used proof-of-work.... proof-of-work is... it has switched to proof-of-stake).